### ISSUE TITLE
[Bug]: Windows Desktop Client does not show folder content of some group folders

### ISSUE THREAD

**Issue by Roeshimi** [nextcloud/desktop#5404]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Some folders inside group folders, e.g. Groupfolder\myfolder\mysubfolder seem to be empty, i.e. the directory structure is there but no files inside those folders are visible. In this example, “myfolder” and “mysubfolder” are there but empty.
This only affects some folders and there’s no visible pattern why this could happening. Also it does not matter if the client is setup to use virtual files or classic file sync. 
It’s no server issue because using different clients (Cyberduck, iOS app), the folders are fine.

### Steps to reproduce

It's not reproducible because I can't say which circumstances are causing the problem. 

### Expected behavior

Files in groupfolder subfolders should be visible 

### Which files are affected by this bug

Files inside some groupfolder subfolders 

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10

### Package

Appimage

### Nextcloud Server version

24.0.9

### Nextcloud Desktop Client version

3.6.6

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 3.3.6 to 3.4.0)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by Roeshimi** [comment]
FWIW if you copy "myfolder" including "mysubfolders" into e.g. myfolder_new, the contents become visible again.

**Comment by Roeshimi** [comment]
Updated to Client version 3.7.1. Still the same issue

**Comment by Roeshimi** [comment]
Updated to version 3.7.3. Issue persists

**Comment by Roeshimi** [comment]
Updated to 3.7.4, Issue persists.

**Comment by Alxprv** [comment]
In version 3.9.1 for windows (tested under Windows 11) the error still exists 
In web - i see files on other language (in my case - russian), in client itsel - too, but in Windows Explorer they do not appear to be. 
When I rename file to english letters, it appears in Windows Explorer

**Comment by Roeshimi** [comment]
> @Roeshimi Is your situation similar to the comment from @Alxprv in any way? (i.e. language makes a difference)
No, we are not using any foreign language characters like Russian, French, etc. Only regular German and English letters. There may be umlauts, but that's rare and also files that don't have any umlauts at all, also do not appear.

**Comment by Sieboldianus** [comment]
I am also seeing this on one of my Windows clients:

1. Local folder is empty, even if I select "Make locally available"

![Image](https://github.com/user-attachments/assets/bb590918-bc65-44b8-9144-3958f5f24e98)

![Image](https://github.com/user-attachments/assets/a5882800-f4a8-4cc7-98e8-342197a855b6)

2. Online, the contents are visible and available

![Image](https://github.com/user-attachments/assets/43a03675-52d9-43a0-a959-cb2f9e633174)

![Image](https://github.com/user-attachments/assets/f77a0259-74e7-4ea3-a483-6eb6e186b3a0)

Other info:

- The folder is inside a group folder/shared folder 
- The shared folder was moved or is inside a regular user documents folder that is synced via virtual files feature
- I have other windows clients where the folder does sync and shows the contents
- I am on Nextcloud 29.0.12, on a stable server for the last 7 years, no other issues
- The client with the issue is on Windows 10

**Comment by Rello** [comment]
Hello,

the client was enhanced significantly since the opening of this ticket.
You can help us with a retest.
If you are still encountering issues, please get back to us
